cmd/tsbs_generate_queries/uses/devops/common.go

```diff
@@ -40,9 +39,6 @@ const (
 	LabelHighCPU = "high-cpu"
 )
 
-// for ease of testing
```

Comment:
I must admit, I did find this ironic when I first read it 😄

Reply:
yeah enjoyed a nice chuckle as I removed it